sonar-db/src/main/java/org/sonar/db/component/ComponentDao.java

```diff
@@ -297,12 +288,16 @@ public void selectForIndexing(DbSession session, @Nullable String projectUuid, R
   private static void addPartialQueryParameterIfNotNull(Map<String, Object> parameters, @Nullable String keyOrNameFilter) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "addPartialQueryParameterIfNotNull" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)